src/api.js

```diff
@@ -57,15 +57,26 @@ class Api {
 	async query( sql, params ) {
 		const _sql = await sql;
 		return this.withConnection( conn => {
-			const data = [];
+			const sets = [];
+			let data;
```

Comment:
How would you feel about renaming `data` to `rows` since the data is an array of sets of rows?

Reply:
I like it.